controllers/storagecluster/cephblockpools.go

```diff
@@ -16,6 +17,8 @@ import (
 
 type ocsCephBlockPools struct{}
 
+var replica1DeviceClasses []rookCephv1.DeviceClasses
```

Comment:
Can we do this without a package var? A local var is preferred. For example, if there are multiple ceph clusters (one internal and one external cluster), there will be confusion about this package var being used for both.

Reply:
When the reconcile request comes for an external storagecluster we are not creating the pools in that case, so the concerned function generateDeviceClass() which would use these variables is never called so there won't be any confusion.